clients/drcachesim/tests/scheduler_launcher.cpp

```diff
@@ -136,10 +140,11 @@ get_current_microseconds()
 
 void
 simulate_core(int ordinal, scheduler_t::stream_t *stream, const scheduler_t &scheduler,
-              std::vector<scheduler_t::input_ordinal_t> &thread_sequence)
+              std::string &thread_sequence)
```

Comment:
Add a comment above documenting what the various characters in the output thread_sequence mean.